partner_credit_limit/model/partner.py

```diff
@@ -58,18 +58,15 @@ def _get_overdue_credit(self):
             # credit = 0.0
             debit_maturity, credit_maturity = 0.0, 0.0
             for line in movelines:
-                if line.date_maturity and line.partner_id.grace_payment_days:
+                limit_day = line.date_maturity
+                if line.partner_id.grace_payment_days:
                     maturity = fields.Datetime.from_string(
                         line.date_maturity)
                     grace_payment_days = timedelta(
                         days=line.partner_id.grace_payment_days)
                     limit_day = maturity + grace_payment_days
                     limit_day = limit_day.strftime("%Y-%m-%d")
```

Comment:
We should use the odoo global variable for this format
Or you could use datetime object directly to avoid format issues.
Something like as:
```python
# Using "limit_day" like as datetime object (not string)
`if (fields.Datetime.from_string(fields.Date.today() - limit_day).days <= 0`